### PR TITLE
* add acknowledgements section [WIP]

### PR DESCRIPTION
**Pull request by tlogan2000**
This is pretty much working but I can't quite figure out how to directly link to `index.html#citation` when starting from a different page (e.g. from `datasets.html`  clicking on the `Citation` option takes me to the index page but I have to click a second time to arrive at `#citation`  ... If someone has a way to do that that would be great otherwise I'll keep googling

fixes https://github.com/Ouranosinc/PAVICS-landing/issues/33

### CODE REVIEW

**Review by aulemahal — COMMENTED** [review]
Pas un expert, mais je pense que ça ne fonctionne pas parce que la page est générée après que le lien http soit résolu. Au moment d'arriver sur la page, le id "citation" n'existe pas encore.

File: src/citation.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+  <head>
+    <!-- NOTE: JUST REPLICATE THIS PAGE AND PUT THE REAL CONTENT IN pages FOLDER -->
+    <title>PAVICS</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" contents="PAVICS is a virtual laboratory facilitating the analysis of climate data. It provides access to several data collections ranging from observations, climate projections and reanalyses. It also provides a Python programming environment to analyze this data without the need to download it. This working environment is constantly updated with the most efficient libraries for climate data analysis, in addition to ensuring quality control on the provided data and associated metadata.">
```
Comment (tlvu) [inline src/citation.html:8]: Probably not breaking anything, just for consistency, the french version do not have this description meta.

**Review by tlvu — APPROVED** [review]
Not a web dev so LGTM.  This will be autodeploy to prod within 1h of merging this PR.

File: src/citation.html (hunk 1 above)
Comment (tlogan2000) [reply src/citation.html]: @aulemahal indicates that this is probably only needed in the english index.html

**Pull request merged**